src/System.Text.Json.Dynamic/System/Text/Json/JsonDynamicObject.cs

```diff
@@ -10,7 +10,17 @@ namespace System.Text.Json
 {
     public class JsonDynamicObject : DynamicObject, IBufferFormattable
     {
-        Dictionary<JsonProperty, JsonValue> _properties;
+        #region Constants
```

Comment:
FYI, this project does not have unsafe enabled (yet?) and I didn't really think it was necessary right now. If you don't like this pattern, I can change it.